app/views/catalog/_bulk_action_button.html.erb

```diff
@@ -1,3 +1,3 @@
 <div class='btn-group'>
-  <%= link_to 'Bulk Actions', bulk_actions_path, class: 'btn btn-outline-secondary', type: 'button' %>
+  <%= link_to 'Bulk Actions', send(:bulk_actions_path, search_params), class: 'btn btn-outline-secondary', type: 'button' %>
 </div>
```

Comment:
pass this search params to bulk actions page when coming from the search results page